tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/npmexporter/VersionConversionFromDeclaredVersionStringTests.xtend

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.npmexporter
+
+import org.eclipse.n4js.N4JSInjectorProvider
+import org.eclipse.n4js.external.version.VersionConstraintFormatUtil
+import org.eclipse.n4js.n4mf.DeclaredVersion
+import org.eclipse.n4js.n4mf.utils.parsing.ManifestValuesParsingUtil
+import java.util.List
+import org.eclipse.xtext.testing.InjectWith
+import org.eclipse.xtext.testing.XtextRunner
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import static org.junit.Assert.*
+
+import static extension org.eclipse.n4js.external.version.VersionConstraintFormatUtil.npmFormat
+import static extension org.eclipse.n4js.n4mf.utils.parsing.ManifestValuesParsingUtil.parseDeclaredVersion
+
+/**
+ * Tests conversion to npm version format done by {@link VersionConstraintFormatUtil}.
+ * Checks are performed for the conversion of {@link DeclaredVersion}. To obtain its instances this tests follow path of parsing
+ * manifest fragments. This is similar what library manager UI is doing to validate user input.
+ * Effectively those tests are also testing some parts of the {@link ManifestValuesParsingUtil}
+ * but it is not focused on it (i.e. not all its execution paths might be tested here).
+ */
+@RunWith(XtextRunner)
+@InjectWith(N4JSInjectorProvider)
+class VersionConversionFromDeclaredVersionStringTests {
```

Comment:
Thanks for finding this one. I removed it since `VersionConversionFromDeclaredVersionStringTest.xtend` is making it obsolete!